src/lib/oauth2-service.ts

```diff
@@ -304,8 +303,6 @@ export class OAuth2Service extends EventEmitter {
     }
 
     assertIsString(redirectUri, 'Invalid redirectUri type');
-    assertIsString(scope, 'Invalid scope type');
```

Comment:
Don't drop this assertion. Although the scope is not being used in the body of this function, the implementation shouldn't accept anything that's not a string (or `undefined`; see my next comment for that).

Reply:
Okay, done.